src/tracker.js

```diff
@@ -1,147 +1,158 @@
-import * as nrvideo from 'newrelic-video-core'
-import {version} from '../package.json'
+import * as nrvideo from "newrelic-video-core";
+import { version } from "../package.json";
 
 export default class ShakaTracker extends nrvideo.VideoTracker {
-  setPlayer (player, tag) {
-    if (!tag && player.getMediaElement) tag = player.getMediaElement()
```

Comment:
My comment here is the same I did for the dash.js tracker: https://github.com/newrelic/video-dash-js/pull/1#discussion_r1768169363

Reply:
For shaka player, Not all events are directly coming from shaka Api. Once we load the Shaka player, then we can use  
method getMediaElement(), by which all html5 events can be tracked.  What We are trying to do is tracking some event which is coming from Shaka from player object and rest from tag object.